blockdag/dag.go

```diff
@@ -1109,7 +1068,12 @@ func (node *blockNode) verifyAndBuildUTXO(dag *BlockDAG, transactions []*util.Tx
 		return nil, nil, nil, nil, err
 	}
 
-	multiset, err = node.calcMultiset(dag, transactions, txsAcceptanceData, selectedParentUTXO, pastUTXO)
+	selectedParentPastUTXO, err := node.selectedParentPastUTXO(dag)
```

Comment:
It makes more sense to just save UTXO diffs as past UTXO in the first place than calling pastUTXO twice.

Reply:
Created a ticket for it.